FREnctools_lib/pyfrenctools/make_mosaic/mosaic_util.py

```diff
@@ -0,0 +1,79 @@
+import ctypes
```

Comment:
this is calling to cfrenctools, it should live in pyfrenctools.  we'll talk about it at the meeting